packages/zipkin-instrumentation-axios/src/index.js

```diff
@@ -1,28 +1,36 @@
 import {Instrumentation} from 'zipkin';
-const zipkinRecordError = (error, options) => {
+
+export const wrapAxios = (axios, options = {}) => {
+  const {tracer} = options;
   const instrumentation = new Instrumentation.HttpClient(options);
-  if (error.response) {
-    instrumentation.recordResponse(options.tracer.id, error.response.status);
-  } else {
-    instrumentation.recordError(options.tracer.id, error);
+  const zipkinRecordRequest = config => tracer.scoped(() => {
+    const newConfig = instrumentation.recordRequest(
+        config,
+        config.url,
+        config.method
+      );
+    newConfig.traceId = tracer.id;
+    return newConfig;
+  });
+  const zipkinRecordResponse = res => tracer.scoped(() => {
+    instrumentation.recordResponse(res.config.traceId, res.status);
```

Comment:
@jcchavezs If here I use `tracer.id` instead of `res.config.traceId`, the tests will fail. I don't know why and I think I don't understand zipkin-js well.